winter/schema/docstring_parser.py

```diff
@@ -0,0 +1,21 @@
+import docstring_parser
+
+
+class DocstringParser:
+
+    def __init__(self, doc: str):
+        docstring = docstring_parser.parse(doc)
+        self.params_docs = {param_doc.arg_name: param_doc for param_doc in docstring.params}
+
+    def get_description(self, argument_name: str, invalid_type_hint: bool = False):
```

Comment:
get_description -> get_argument_description
Because docstring contains not only arguments description.